src/app/tests/suites/certification/Test_TC_DGWIFI_2_1.yaml

```diff
@@ -56,7 +56,7 @@ tests:
           constraints:
               type: enum8
               minValue: 0
-              maxValue: 5
+              maxValue: 6
```

Comment:
Confirmed that test plan just says "The data type must match the value listed in the specification(s) WiFiVersion ENUM table.", which would be at most 6.